### PR TITLE
Update conprof &prof.md

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8084, https://github.com/pingcap/docs-cn/pull/8204
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-faq.md [hunk 1]
```diff
@@ -80,3 +80,59 @@ To clear your browser cache, take the following steps:
 3. Select the **Application** panel, expand the **Local Storage** menu and select the **TiDB Dashboard page domain**. Click the **Clear All** button.
 
     ![Clear the Local Storage](/media/dashboard/dashboard-faq-devtools-application.png)
+
+### NgMonitoring does not start as expected
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:84]:
```suggestion
### A `required component NgMonitoring is not started` error notification is shown
```

File: dashboard/dashboard-faq.md [hunk 2]
```diff
@@ -80,3 +80,59 @@ To clear your browser cache, take the following steps:
...
+
+If the **Continuous Profiling** page shows that NgMonitoring does not start, perform the following steps to address the problem.
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:86]:
```suggestion
If the **Continuous Profiling** page shows that NgMonitoring component is not started, perform the following steps to address the problem.
```

**Review by breeswish — COMMENTED** [review]
Cool! The rest looks good.

File: dashboard/dashboard-profiling.md [hunk 3]
```diff
@@ -8,11 +8,15 @@ aliases: ['/docs/dev/dashboard/dashboard-profiling/']
...
+## Profiling content
+
+With Manual Profiling, you can collect continuous performance data of TiDB, PD, TiKV, and TiFlash instances. The data collected can be displayed in various forms, for example, on a flame graph or a directed acyclic graph. The data displayed visually shows what internal operations are performed on the instances during the performance profiling period and the corresponding proportions. With such data, you can quickly learn the CPU resource consumption of these instances.
+
 ## Access the page
 
 You can access the instance profiling page using either of the following methods:
 
-- After logging into TiDB Dashboard, click **Advanced Debugging** > **Profile Instances** > **Manually Profile** on the left navigation bar.
+- After logging into TiDB Dashboard, click **Advanced Debugging** > **Profiling Instances** > **Manual Profiling** on the left navigation bar.
```
Comment (shichun-0415) [inline dashboard/dashboard-profiling.md:19]: @breeswish The figure in Line 21 needs updating. Thanks.

File: dashboard/continuous-profiling.md [hunk 4]
```diff
@@ -18,111 +18,44 @@ With less than 0.5% performance loss, this feature takes continuous snapshots (s
...
 Currently, Continuous Profiling can display the following performance data:
 
 - TiDB/PD: CPU profile, Heap, Mutex, Goroutine (debug=2)
-- TiKV: CPU profile
+- TiKV/TiFlash: CPU profile
 
 ## Enable Continuous Profiling
 
-To enable Continuous Profiling, you need to first check the version information and then configure related settings on the control machine and TiDB Dashboard.
+This section describes how to enable Continuous Profiling on TiDB clusters deployed using TiUP and TiDB Operator respectively.
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply dashboard/continuous-profiling.md:36]: Fixed in https://github.com/pingcap/docs-cn/pull/8204

**Review by qiancai — APPROVED** [review]

**Review by breeswish — APPROVED** [review]

**Comment by breeswish** [comment]
Thanks a lot!

**Pull request merged**